pkg/scheduler/metrics/metrics.go

```diff
@@ -187,6 +187,15 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"type"})
 
+	PodPriority = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      SchedulerSubsystem,
+			Name:           "pod_priority",
```

Comment:
while I don't have any strong opinions on whether this should be a new metric on its own, perhaps we could explore the possibility of adding pod priority as a label to existing metrics such as `pod_scheduling_attempts` or `pod_scheduling_duration_seconds`. However, one concern will be the high cardinality of pod priorities.

Reply:
Thinking a bit more on this I think a counter metrics to keep count of how many pods have priority>0 maybe more useful..what's your thoughts @ahg-g ?

Reply:
I fear that adding pod priority to existing metrics would make the cardinality explode. I think having this separate metric makes sense.